src/playbin.js

```diff
@@ -0,0 +1,157 @@
+/* exported Playbin */
+
+const {Gdk, GLib, GObject, Gtk, Gst} = imports.gi;
+
+var Playbin = GObject.registerClass({
+    CssName: 'playbin',
+    Properties: {
+        uri: GObject.ParamSpec.string('uri', 'URI', '',
+            GObject.ParamFlags.READWRITE,
+            ''
+        ),
+    },
+    Signals: {
+        clicked: {},
+        start: {},
+        done: {},
+    },
+}, class Playbin extends Gtk.EventBox {
+    _init(props) {
+        super._init(props);
+
+        this._style = this.get_style_context();
```

Comment:
There's no need to cache this as a private member in JS, since GTK already does that for us.